test/make.sh

```diff
@@ -73,12 +83,25 @@ function docker() {
 # This function runs 'terraform validate' against all
 # directory paths which contain *.tf files.
 function check_terraform() {
-  echo "Running terraform validate"
+  local rval=125
+  # fmt is before validate for faster feedback, validate requires terraform
+  # init which takes time.
+  echo "Running terraform fmt"
   find_files . -name "*.tf" -print0 \
```

Comment:
Perhaps better and simpler will be a checklist of files,  not directories even if it will be directories, we could simplify construction, with minimization quantity of bash calls so it will be enough `find . -type f -name "*.tf" -exec terraform fmt  -diff -check=true -write=false {} \;`

Reply:
This optimization is out of scope of the migration and to be addressed [here](https://github.com/terraform-google-modules/terraform-google-project-factory/issues/250)